pulsar-io/elastic-search/src/test/java/org/apache/pulsar/io/elasticsearch/ElasticSearchSinkRawDataTests.java

```diff
@@ -64,22 +62,13 @@
 
     @BeforeClass
     public static final void initBeforeClass() {
-        container = createElasticsearchContainer();
```

Comment:
what about creating two classes instead of start/stopping the container for every test method ?

you can add the image name to the constructor and create a "Base" class,
this way you are not going to change the tests at all, only simply adding two subclasses of this base class